explore/src/main/scala/explore/config/ImagingConfigurationPanel.scala

```diff
@@ -114,6 +128,37 @@ object ImagingConfigurationPanel {
                     }).runAsyncAndForgetCB
                   )
                   .getOrElse(Callback.empty)
+            ),
+            <.label("Field of View", HelpIcon("configuration/fov.md"), ExploreStyles.SkipToNext),
+            InputWithUnits(
+              id = "configuration-fov",
+              clazz = Css.Empty,
+              inline = true,
+              value = fov,
+              units = "arcsec",
+              validFormat = ValidFormatInput.fromFormatOptional(formatArcsec),
+              changeAuditor = ChangeAuditor.fromFormat(formatArcsec).optional,
+              disabled = false
+            ),
+            <.label("S / N",
+                    HelpIcon("configuration/signal_to_noise.md"),
+                    ExploreStyles.SkipToNext
+            ),
+            FormInputEV(id = "signal-to-noise",
+                        value = signalToNoise,
+                        validFormat = ValidFormatInput.fromFormatOptional(formatPosBigDecimal)
```

Comment:
No change auditor desired?

Reply:
You're right, we should have a `ChangeAuditor` defined here